src/router/index.js

```diff
@@ -22,9 +24,11 @@ const AppRouter = () => {
         <Route path="/signup">
           <SignUp />
         </Route>
-        <Route path="/game">
-          <Game />
-        </Route>
+        <PrivateRoute>
+          <Route path="/game">
+            <Game />
+          </Route>
+        </PrivateRoute>
```

Comment:
Nit: 개인적인 생각인데 `<PrivateRoute />`는 결국에는 Route이기 때문에 다음과 같은 형태를 가지는건 어떤가요?

```suggestion
<PrivateRoute path="/game">
    <Game />
</PrivateRoute>
```